testsuite/features/build_validation/init_clients/centos7_minion.feature

```diff
@@ -4,29 +4,29 @@
 #  1) bootstrap a new CentOS 7 minion via salt-ssh
```

Comment:
This is not via salt-ssh, right? It's just webui without the ssh option checked. Please also change this

Reply:
Thanks for fixing this extra thing